docs/readthedocs/models/em.md

```diff
@@ -0,0 +1,38 @@
+(em)=
+
+# Energy Management
+
+Energy Management Agents (EmAgents) control power feed-in and load of system participants or other EmAgents in a hierarchical fashion.
+
+## Protocol
+
+During simulation, EmAgents send `RequestFlexOptions` and `IssueFlexControl` messages and receive `ProvideFlexOptions` and `FlexCompletion` messages.
+After having been requested to calculate flex options via `RequestFlexOptions`, controllable assets send back their flex options to the controlling unit using `ProvideFlexOptions`. 
+Eventually the controlling EmAgent responds with some type of `IssueFlexControl` messages, setting a power set point for operation.
+The asset then tries to realize the set power as best as it can and replies with a `FlexCompletion` messages.
+If an EmAgent is itself controlled by another EmAgent, it also behaves like a system participant (sends `RequestFlexOptions` and `IssueFlexControl` messages etc.).
+
+
+Every EmAgent aggregates flex options and power of its connected assets and disaggregates flex control among the connected assets. 
+It also functions as a scheduler for all connected assets by processing information on the ticks of the next desired activations and conveying such information to the controlling EmAgent or a central scheduler respectively.
+
+Uncontrolled EmAgents answer to a scheduler with regards to their activation.
+
+![](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/ie3-institute/simona/dev/docs/uml/protocol/em/UncontrolledEm.puml)
```

Comment:
The plantuml implementation isn't rendering.

Reply:
It'll work once the diagrams are in `dev`, because we're taking the sources from there

Reply:
Make sense, thanks for clarification :)